docs/guides/sdk/authentication.md

```diff
@@ -0,0 +1,184 @@
+---
+contributors: Kevin Lewis, Tim de Heiden
+description: Learn about all of the ways to control authentication with the Directus SDK
+---
+
+# Directus SDK Authentication
+
+The Directus SDK provides multiple ways to [authenticate your requests](/reference/authentication), by either generating
+an access and refresh tokens through login, or by setting a static token.
```

Comment:
"by either generating access and refresh tokens..."

The sentence is fine without the "an"

Reply:
Nice catch! 👍 Fixed now